### PR TITLE
feat: upload images on markdown

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes

File upload by dropping files on the textarea or clicking the footer of the textarea itself.

The process flow is taken from the current behavior of GitHub's markdown input support for files. The upload process can be done with multiple files but each file is processed one after the other.

Have refactored to make the instance of the command class be created only once.

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1336 #done

### CODE REVIEW

File: packages/shared/src/components/fields/ImageInput.tsx [hunk 1]
```diff
@@ -48,7 +48,8 @@ const sizeToIconSize: Record<Size, IconSize> = {
   medium: IconSize.Small,
   large: IconSize.Medium,
 };
-const ACCEPTED_TYPES = 'image/png,image/jpeg';
+export const ACCEPTED_TYPES = 'image/png,image/jpeg';
```
Comment (idoshamun) [inline packages/shared/src/components/fields/ImageInput.tsx:51]: gif support will come later?

File: packages/shared/src/components/fields/MarkdownInput/index.tsx [hunk 2]
```diff
@@ -46,8 +53,47 @@ function MarkdownInput({
     initialContent,
     onSubmit,
     textareaRef,
+    enableUpload,
   });
 
+  const footerLabel = (() => {
```
Comment (idoshamun) [inline packages/shared/src/components/fields/MarkdownInput/index.tsx:59]: Let's perhaps create a custom component for this? Will it make more readable

**Review by idoshamun — CHANGES_REQUESTED** [review]
Let's also support pasting an image to upload. It's also a common practice

File: packages/shared/src/components/fields/ImageInput.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/fields/ImageInput.tsx:51]: Should be on the same release, let me add it as custom only on the areas where it is needed.

**Review by idoshamun — APPROVED** [review]
Looks amazing!!

**Pull request merged**